features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -50,29 +50,42 @@ internal class Base64Serializer private constructor(
         applicationContext: Context,
         displayMetrics: DisplayMetrics,
         drawable: Drawable,
+        drawableWidth: Int,
+        drawableHeight: Int,
         imageWireframe: MobileSegment.Wireframe.ImageWireframe
     ) {
         registerCacheForCallbacks(applicationContext)
         registerBitmapPoolForCallbacks(applicationContext)
 
         asyncImageProcessingCallback?.startProcessingImage()
 
-        var shouldCacheBitmap = false
         val cachedBase64 = base64LRUCache?.get(drawable)
         if (cachedBase64 != null) {
             finalizeRecordedDataItem(cachedBase64, imageWireframe, asyncImageProcessingCallback)
             return
         }
 
+        var shouldCacheBitmap = false
+
```

Comment:
I don't see where is the unit test related with this fix

Reply:
There are tests for the bitmapDrawable scaling [here](https://github.com/DataDog/dd-sdk-android/blob/jmoskovich/rum-1012/fix-rippledrawables/features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64SerializerTest.kt#L467)